### DISCUSSION TITLE
Getting started docs in text?

### DISCUSSION THREAD

**Discussion by brimwats1** [marcusolsson/obsidian-vale#15] [Q&A]
Hello! I was curious about trying out vale, but I am deaf and when i clicked into the video on Vimeo I didn't see captions. Are these available somewhere, or is the text of the video available?

**Comment by marcusolsson** [comment]
There's only a music track on the video, so you're not missing out on much :) I've been meaning to create a written guide, but I haven't found the time yet. Let me know if there's anything I can help clarify!
